common/lib/xmodule/xmodule/js/fixtures/tabs-edit.html

```diff
@@ -41,10 +41,10 @@ <h2 class="title modal-window-title">Mock Modal Title</h2>
             <div class="modal-actions">
                 <ul>
                     <li class="action-item">
-                        <a href="#" class="button action-primary action-save">Save</a>
+                        <button type="button" class="button action-primary action-save">Save</button>
```

Comment:
But you shouldn't change fixture files if they don't reflect the actual HTML. Can you please take another look (I don't see how this is related to CMS vs. LMS)?